app/controllers/ngos/ongoing_events_controller.rb

```diff
@@ -0,0 +1,74 @@
+class Ngos::OngoingEventsController < ApplicationController
+  before_action :authenticate_ngo!
+
+  def index
+    @events =
+      OngoingEventOperation::Index.present(current_ngo: current_ngo).model
+  end
+
+  def show
+    @event =
+      OngoingEventOperation::Show.present(current_ngo: current_ngo, event_id: params[:id]).model
+  end
+
+  def new
+    @event =
+      OngoingEventOperation::Create.present(current_ngo: current_ngo).model
+  end
+
+  def create
+    @event = OngoingEventOperation::Create.(
+      current_ngo: current_ngo,
+      ongoing_event: event_params(params)
+    ).model
+    if @event.valid?
+      return redirect_to ngos_ongoing_events_url,
+        notice: t("ngos.ongoing_events.messages.create_success")
+    end
+    render :new
+  end
+
+  def edit
+    @event =
+      OngoingEventOperation::Update.present(current_ngo: current_ngo, event_id: params[:id]).model
+  end
+
+  def update
+    @event =
+      OngoingEventOperation::Update.(
+        current_ngo: current_ngo,
+        event_id: params[:id],
+        ongoing_event: event_params(params)
+      ).model
+    if @event.valid?
```

Comment:
Same as for `create`: do you even reach this statement when passing invalid `event_params`?